Applications/spire/source/ui/drop_shadow.cpp

```diff
@@ -1,8 +1,10 @@
 #include "spire/ui/drop_shadow.hpp"
+#include <QApplication>
 #include <QEvent>
 #include <QLinearGradient>
 #include <QPainter>
 #include <QRadialGradient>
+#include "qt_windows.h"
```

Comment:
Where is this header found?

Reply:
What's wrong with it? Should it be <qt_windows.h>?

Reply:
Yes.